pymodels/SI_V25_02/lattice.py

```diff
@@ -0,0 +1,865 @@
+
+    # --- IDB insertion sectors ---
+
+    IDB = [
+        L500, LIB, L500, MIDB, L500, L500, MIB, L500, L500, MIDB, L500, LIB,
+        L500]  # low beta ID straight section
+    
+    IDB_GSL07 = [
+        L500, GSL07, LIB, L500,
+        MIDB, L500, L500, MIB, L500, L500, MIDB,
+        L500, LIB, L500]  # low beta ID straight section
+    
+    IDB_TunePkup = [
+        L500, LIB, L500,
+        MIDB, L500, L500, MIB, L500, L500, MIDB,
+        L500, TunePkup, LIB, L500]  # low beta ID straight section
+    
+    IDB_02 = [
+        L500, LIB, L500,
+        MIDB, L500, L500, MIB, L500, L500, MIDB,
+        L500, HCav, LIB, L500]  # low beta ID straight section
+    
+    IDB_06 = [
+        L500, LIB, L500,
+        L350, MIDB, ID06H, MIB, ID06H, MIDB,
+        L350, L500, LIB, L500]  # low beta ID straight section (CARNAUBA)
+
+    IDB_08 = [
+        L500, LIB, L500, L350, 
+        MIDB, ID08H, MIB, ID08H, MIDB,
+        L350, L500, LIB, L500]  # low beta ID straight section (EMA)
+
+    IDB_10 = [
+        L500, LIB, L665, IDC, L135,
+        MIDB, ID10H, MIB, ID10H, MIDB,
+        L135, IDC, L665, LIB, L500]  # low beta ID straight section (SABIA)
+    
+    IDB_12 = [
+        L500, LIB, L665, L100, L135,
+        MIDB, ID12H, MIB, ID12H, MIDB,
+        L135, L100, L665, LIB, L500]  # low beta ID straight section (COLIBRI)
+
+    IDB_16 = [
+        L500, LIB, L500,
+        MIDB, L500, L500, MIB, L500, L500, MIDB,
+        L500, BbBKckL, LIB, L500]  # low beta ID straight section (INGA)
+
+    # --- IDP insertion sectors ---
+
+    IDP = [
+        L500, LIP, L500,
+        MIDP, L500, L500, MIP, L500, L500, MIDP,
+        L500, LIP, L500]  # low beta ID straight section
+    
+    IDP_CAV = [
+        L500, LIP, L500, L500, L500, MIP, RFC, L500, L500, L500,
+        LIP, L500]  # low beta RF cavity straight section
+    
+    IDP_07 = [
+        L500, LIP, L500, L350,
+        MIDP, ID07H, MIP, ID07H, MIDP,
+        L350, L500, LIP, L500]  # low beta ID straight section (CATERETE)
+        
+    IDP_11 = [
+        L500, LIP, L500, L350,
+        MIDP, ID11H, MIP, ID11H, MIDP,
+        L350, L500, LIP, L500]  # low beta ID straight section (IPE)
+
+    IDP_GSL15 = [
+        L500, GSL15, LIP, L500,
+        MIDP, L500, L500, MIP, L500, L500, MIDP,
+        L500, LIP, L500]  # low beta ID straight section
+
+
+    # -- girders --
+
+    # straight sections
+    SS_S01 = IDA_INJ  # INJECTION
+    SS_S02 = IDB_02
+    SS_S03 = IDP_CAV
+    SS_S04 = IDB
+    SS_S05 = IDA_ScrapH
+    SS_S06 = IDB_06  # CARNAUBA
+    SS_S07 = IDP_07  # CATERETE
+    SS_S08 = IDB_08  # EMA
+    SS_S09 = IDA_09  # MANACA
+    SS_S10 = IDB_10  # SABIA
+    SS_S11 = IDP_11  # IPE
+    SS_S12 = IDB_12  # COLIBRI
+    SS_S13 = IDA_BbBKckrH
+    SS_S14 = IDB  # HARPIA
+    SS_S15 = IDP  # SAGUI
+    SS_S16 = IDB_16  # INGA
+    SS_S17 = IDA_17  # SAPUCAIA
+    SS_S18 = IDB_TunePkup  # PAINEIRA
```

Comment:
The beamline information regarding the corresponding straight section was recently updated [here](https://wiki-sirius.lnls.br/mediawiki/index.php/Machine:Insertion_Devices#Beamline_Insertion_Devices_Specifications) 

The difference from [this table](https://wiki-sirius.lnls.br/mediawiki/index.php/Machine:Storage_Ring#Straight_sections_allocation) is that PAINERA 18SB -> 14SB (replacing HARPIA)

COLIBRI, HARPIA, SAGUI and INGA beamlines are not mentioned in the new table.

Reply:
I updated this PR according to the wiki table. thanks!